MANUAL.md

```diff
@@ -1,239 +0,0 @@
-CVE checker for binary code User Manual
```

Comment:
Looks like you accidentally removed MANUAL.md and README.md entirely in this PR.  That's going to make this code un-mergeable until it's fixed.  You might have to learn a little bit of git to figure it out, or if you get stuck with the git commands to get the original file from master and put it back, you can also copy your changes to a new branch off the latest master tree and just don't touch the .md files.